packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -986,11 +986,29 @@ to configure these.
 There are a number of limitations in using OpenAPI definitions in API Gateway. Read the [Amazon API Gateway important
 notes for REST APIs](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-known-issues.html#api-gateway-known-issues-rest-apis)
 for more details.
-
+ 
 **Note:** When starting off with an OpenAPI definition using `SpecRestApi`, it is not possible to configure some
 properties that can be configured directly in the OpenAPI specification file. This is to prevent people duplication
 of these properties and potential confusion.
 
+### Endpoint configuration
+
+By default, SpecRestApi will create an endpoint with a edge optimized endpoint configuration.
+
+It is possible to modify this by providing the endpointConfiguration as shown here:
+
+```ts
+const api = new apigateway.SpecRestApi(this, 'ExampleRestApi', {
+  apiDefinition: apigateway.ApiDefinition.fromInline(replacedSwagger),
+  endpointConfiguration: {
+    types: [apigateway.EndpointType.PRIVATE],
+  },
```

Comment:
Is it not possible to configure endpoint types in the OpenAPI spec directly?

Reply:
couldn't find a way. I tried using parameters and checked all the [openAPI extensions](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions.html) but couldn't find.

I thought about checking for the [x-amazon-apigateway-endpoint-configuration](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-endpoint-configuration.html) and if there is a VPC configure the endpoint as private, but that means you can not then configure regional ones (so decided against that approach). 

It will be great is that x-amazon-apigateway-endpoint-configuration had also the type...